.github/workflows/ci.yml

```diff
@@ -14,6 +17,8 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: php:${{ matrix.php }}-alpine
+      options: >-
+        --tmpfs /tmp:exec
```

Comment:
what does this speed up?

Reply:
I think the main thing that made the Behat tests really slow was slow disk I/O for sqlite. I'm not sure why...

Reply:
nice